src/theory/arith/theory_arith_private.cpp

```diff
@@ -3938,15 +3938,36 @@ Node TheoryArithPrivate::branchIntegerVariable(ArithVar x) const {
   TNode var = d_partialModel.asNode(x);
   Integer floor_d = d.floor();
 
-  //Node eq = Rewriter::rewrite(NodeManager::currentNM()->mkNode(kind::EQUAL, var, mkRationalNode(floor_d+1)));
-  //Node diseq = eq.notNode();
+  if (options::cubeTest())
+  {
+    Trace("integers") << "cube test enabled"  << endl;
+    Integer ceil_d = d.ceiling();
+    Rational f = r - floor_d;
+    // Multiply by -1 to get abs value.
+    Rational c = (r - ceil_d) * (-1); 
```

Comment:
simplified: `c = ceil_d - r`

Reply:
` error: no match for ‘operator-’ (operand types are ‘CVC4::Integer’ and ‘const CVC4::Rational’)
     Rational c = ceil_d - r;
                  ~~~~~~~^~~
`